src/components/expression/expression-renderer.js

```diff
@@ -20,40 +20,16 @@ genotet.ExpressionRenderer = function(container, data) {
    *   container_: d3.selection,
    *   geneName: string,
    *   row: number,
-   *   color: string
+   *   hoverColumn: number,
+   *   hoverConditionName: string,
+   *   hoverValue: number,
+   *   color: string,
+   *   clicked: bool
    * }>}
    */
   this.data.profiles;
```

Comment:
"Now that you typedef-ed Profile, you can use your defined type for this." 
Should this be !Array<!genotet.ExpressionRenderer.Profile>?